### DISCUSSION TITLE
Using WixUI to install to network drive

### DISCUSSION THREAD

**Discussion by valitana-jlim** [wixtoolset/issues#7441] [Questions] [answered]
Hello, 

One of our clients faced issues trying to install our application to their network drive. I've seen that some people have asked about this particular problem online, but I wasn't sure if things had changed for wix v4. Is it possible for users to install applications into a shared network drive?

Thank you!

**Comment by robmen** [comment, marked as answer]
The Windows Installer (not WiX) is a system service designed to operate on the local machine. You can provide it access to network drives and make these sorts of things kinda' work, but the package gets registered on the local machine. You'll likely encounter challenging edge cases. I'd recommend a different design over this one.
